Author: Russell Coker <russell@coker.com.au>
Description: The following patch adds a section to the libFLAC.so file to indicate that
it doesn't need an executable stack. This allows programs using it to run
correctly on SE Linux systems, and with exec-shield.
--- flac.orig/src/libFLAC/ia32/nasm.h
+++ flac/src/libFLAC/ia32/nasm.h
@@ -73,3 +73,6 @@
 _%1:
 %1:
 %endmacro
+
+section .note.GNU-stack progbits noalloc noexec nowrite align=1
+
